api/CustomEvent.json

```diff
@@ -0,0 +1,241 @@
+{
+  "api": {
+    "CustomEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CustomEvent",
+        "support": {
+          "webview_android": {
+            "version_added": true,
+            "notes": "Available in workers"
```

Comment:
Worker support is done the same as a subfeature because sometimes it has a different version number than the feature itself. Here's an example https://github.com/mdn/browser-compat-data/blob/master/api/WebGLBuffer.json